config/config.go

```diff
@@ -74,11 +74,12 @@ func getHomeDir() string {
 	return homeDir
 }
 
+// Load the config from file path provided, read from default path if configFilePath is empty
 func Load(configFilePath string) (Config, error) {
 
 	config := Config{}
 
-	body, readErr := ioutil.ReadFile(configFilePath)
+	body, readErr := ioutil.ReadFile(getConfigurationFile(configFilePath))
```

Comment:
Do we need this line in configManager now? https://github.com/deislabs/ratify/blob/240aabceb4f8ec62c3a2c31ef29131bb7fdda5c5/config/configManager.go#L44

Reply:
Yes, this is stil needed. The config manager needs to know the exact file path to add the file watcher on the correct file.

Reply:
oh yep I see. `configFilePath` is used further down for the watcher